US County Data - pandas.ipynb

```diff
@@ -7,7 +7,10 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    Q1 = united_States_covid.reset_index()
</pre><p>Do not need this reset_index</p><p> </p><p>New York City / New York is not a county, hence the FIPS code is NaN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CMU-MS-DAS-Modern-Programming-Mini/ClassAssignmentNo4/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>